### ISSUE TITLE
Getting TypeScript error in `node_modules/graphql-tag/lib/index.d.ts`(1,13): '=' expected.  TS1005

### ISSUE THREAD

**Issue by ErnestoBorio** [apollographql/graphql-tag#345]
I'm getting `TypeScript error in /app/node_modules/graphql-tag/lib/index.d.ts(1,13): '=' expected.  TS1005` when building my app, which depends on `graphql-tag@^2.11.0` via `@apollo/client@^3.3.2`.

```
TypeScript error in /app/node_modules/graphql-tag/lib/index.d.ts(1,13):
'=' expected.  TS1005

  > 1 | import type { DocumentNode } from 'graphql/language/ast';
      |             ^
```

**Comment by krishnabhargav** [comment]
Getting the same issue. Seems like the 2.11 version was replaced by accident? Anyway, we resolved out issue by upping the typescript version to be 3.8+.

**Comment by benjamn** [comment]
@ErnestoBorio @krishnabhargav Sorry about this problem, but thank you for reporting it! It was introduced by https://github.com/apollographql/apollo-client/pull/7651 (in `@apollo/client@3.3.9`), which I considered reverting, but I believe PR #352 solves the problem without rolling anything back. Those changes are available now in `graphql-tag@2.12.1`.
